src/views/undo.h

```diff
@@ -20,24 +20,16 @@
 #define DT_UNDO_H
 
 #include "common/dtpthread.h"
-#include "views/view.h"
 #include <glib.h>
 
 //  types that are known by the undo module
 typedef enum dt_undo_type_t
 {
   DT_UNDO_GEOTAG = 1,
+  DT_UNDO_HISTORY = 2,
 } dt_undo_type_t;
 
-#define DT_UNDO_ALL (DT_UNDO_GEOTAG)
-
-//  types that are known by the undo module are declared here to be shared by
-//  all the views supporting undo.
-typedef struct dt_undo_geotag_t
-{
-  int imgid;
-  float longitude, latitude, elevation;
-} dt_undo_geotag_t;
+#define DT_UNDO_ALL (DT_UNDO_GEOTAG | DT_UNDO_HISTORY)
```

Comment:
Hm, then it should be like this:
```
typedef enum dt_undo_type_t
{
  DT_UNDO_GEOTAG = 1 << 0,
  DT_UNDO_HISTORY = 1 << 1,
  DT_UNDO_ALL = DT_UNDO_GEOTAG | DT_UNDO_HISTORY
} dt_undo_type_t;
```
I'd be surprised to learn that it does not work...